### DISCUSSION TITLE
No Regions to Draw in Batch Processing Open Field videos

### DISCUSSION THREAD

**Discussion by jocelynasp** [DeniseCaiLab/ezTrack#68] [Q&A]
I'm trying to do batch processing for some habituation videos where there is only one mouse in the chamber and nothing else. However, when I go to draw the regions of interest, it says no regions to draw and I don't get the icon that I get with individual processing to draw the regions.

<img width="589" alt="image" src="https://user-images.githubusercontent.com/99834393/217043261-5871d9d4-bda1-4582-883e-18cfee60261f.png">

**Comment by ZachPenn** [comment]
I'm guessing you didn't define the names of the regions of interest in the second cell, where other parameters are defined.
